### PR TITLE
Created diagrams for DG.

### PR DESCRIPTION
**Pull request by omupenguin**

### CODE REVIEW

**Review by yetong1895 — CHANGES_REQUESTED** [review]
there is a typo at the name of ArchitectureSequenceDi"s"gram.puml

**Comment by omupenguin** [comment]
I'm not sure what typo you're referring to, can you elaborate more?

**Comment by yetong1895** [comment]
> I'm not sure what typo you're referring to, can you elaborate more?

docs/diagrams/ArchitectureSequenceDisgram.puml
There is typo in Diagram.

File: docs/diagrams/ArchitectureSequenceDisgram.puml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@startuml
```
Comment (Weng-Kexin) [inline docs/diagrams/ArchitectureSequenceDisgram.puml]: @omupenguin , yetong1895 meant the naming of this puml file

Comment (omupenguin) [reply docs/diagrams/ArchitectureSequenceDisgram.puml]: oh, thanks for the catch!

**Review by Weng-Kexin — APPROVED** [review]

**Review by yetong1895 — APPROVED** [review]

**Review by tatayu — APPROVED** [review]

**Pull request merged**